src/components/WebexRemoteMedia/WebexRemoteMedia.jsx

```diff
@@ -22,16 +25,37 @@ export default function WebexRemoteMedia({className, meetingID}) {
     remoteShare,
     error,
   } = useMeeting(meetingID);
+  const members = useMemberships(meetingID, 'meeting');
   const audioRef = useStream(remoteAudio);
   const videoRef = useStream(remoteVideo);
   const shareRef = useStream(remoteShare);
+  const hasOtherMembers = members.length > 1;
   const hasMedia = !!(remoteAudio || remoteVideo || remoteShare);
   const hasTwoMedia = remoteVideo && remoteShare;
   const mainClasses = {
     [`${WEBEX_COMPONENTS_CLASS_PREFIX}-remote-media`]: true,
     [className]: !!className,
     'split-screen': hasTwoMedia,
   };
+  let remoteDisplay;
+
+  if (hasMedia) {
+    remoteDisplay = hasOtherMembers ? (
+      <>
+        {remoteVideo ? <video ref={videoRef} playsInline autoPlay /> : null}
+
+        {remoteShare ? <video ref={shareRef} playsInline autoPlay /> : null}
+
+        {remoteAudio ? <audio ref={audioRef} autoPlay /> : null}
+      </>
+    ) : (
+      <>
+        <Badge rounded>
```

Comment:
This still has the badge, and so it renders like this:
![with badge](https://user-images.githubusercontent.com/6610308/91893451-fb87c180-ec48-11ea-82f6-9d4cc5bdebab.png)

This is what the UI in the client is, which, I'm totally guessing here, but looks like an [`h4` from Momentum typography](https://momentum.design/styles/typography/style)?
![client](https://user-images.githubusercontent.com/6610308/91893572-2e31ba00-ec49-11ea-946c-1eb960921135.png)